property_tests/arbitraries/candid/candid_definition_arb/types.ts

```diff
@@ -59,16 +61,11 @@ export type ServiceCandidDefinition = {
     candidMeta: CandidMeta;
     funcs: ServiceMethodDefinition[];
 };
-export type ServiceMethodDefinition = {
-    name: string;
-    imports: Set<string>;
-    variableAliasDeclarations: string[];
-    src: string;
-};
 
 type CandidMeta = {
     candidTypeAnnotation: string; // Either a type reference or type literal
     candidTypeObject: string;
+    azleCandidTypeObject: AzleCandidType;
```

Comment:
Rename azleCandidTypeObject, AzleCandidType type seems like it has a weird name or type...